bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -308,14 +305,14 @@ void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
     batchingFactor = (maxNumberOfPendingRequestsInRecentHistory / aa);
     if (batchingFactor < 1) batchingFactor = 1;
     maxNumberOfPendingRequestsInRecentHistory = 0;
+    LOG_DEBUG(GL, "PrePrepare batching factor updated. " << KVLOG(batchingFactor));
   }
 
   // because maxConcurrentAgreementsByPrimary <  MaxConcurrentFastPaths
   Assert((primaryLastUsedSeqNum + 1) <= lastExecutedSeqNum + MaxConcurrentFastPaths);
 
   CommitPath firstPath = controller->getCurrentFirstPath();
 
-  // assert: (cVal==0) --> (firstPath != CommitPath::FAST_WITH_THRESHOLD)
   Assert((config_.cVal != 0) || (firstPath != CommitPath::FAST_WITH_THRESHOLD));
```

Comment:
Maybe use AssertOR here?

Reply:
I'm going to create a separate PR that changes all the asserts to use the new forms.